.gitlab-ci.yml

```diff
@@ -3,16 +3,13 @@ image: mcr.microsoft.com/dotnet/core/sdk:3.1
 # Original URL format for agent artifact:
 # http://s3.amazonaws.com/dsd6-staging/windows/agent7/buildpack/agent-binaries-7.25.0-1-x86_64.zip
 
-# Original URL format for windows tracer home:
-# https://github.com/DataDog/dd-trace-dotnet/releases/download/v1.21.1/windows-tracer-home.zip
-
 variables:
   DOTNET_AGENT_DOWNLOAD_URL: "http://apm-dotnet-throughput-cannon.eastus.cloudapp.azure.com/Builds/Custom/NamedPipesAAS/v20210108/agent-binaries-named-pipes-7.24.0.git.2.49f4f6c-1-x86_64.zip" 

```

Comment:
Is there an official agent yet with the named pipes support or do we have to use this pre-release version?

Reply:
Not officially released yet, but this was the named pipes cherry picked and built on a stable version.